docs/core/development/policy/compatibility-policy.rst

```diff
@@ -132,8 +132,6 @@ By doing so, there's no chance anything could access these objects by going thro
 
 Test code and test helpers are considered private API and should not be imported outside
 of the Twisted testing infrastructure.
-As an exception to this, :api:`twisted.test.proto_helpers` is considered a public API
-(see `#6435 <https://twistedmatrix.com/trac/ticket/6435>`_ for more discussion).
```

Comment:
Did this thinking on this change? Seems unrelated to the topic…

Reply:
@wsanchez https://freenode.logbot.info/twisted-dev/20201015#c5467846

Reply:
okie dokie